.prettierignore

```diff
@@ -0,0 +1,5 @@
+*.git
+*.svn
+*.hg
+
+node_modules
```

Comment:
Is this working fine for `node_modules`? Can you try going into any file inside the `node_modules` and formatting it? If it formats it means it's not working and vice-versa.

Reply:
yes node_modules are not formatting